docs/content/latest/admin/yb-admin.md

```diff
@@ -892,16 +892,46 @@ yb-admin restore_snapshot_schedule <schedule-id> <restore-target>
 ```
 
 * _schedule-id_: The identifier (ID) of the schedule to be restored.
-* _restore-target_: The time to which to restore the snapshots in the schedule. This can be either an absolute Unix time, or a relative time such as `minus 5m` (to restore to 5 minutes ago). Optional; omit to restore to each snapshot's creation time.
+* _restore-target_: The time to which to restore the snapshots in the schedule. This can be either an absolute Unix timestamp, or a relative time such as `minus 5m` (to restore to 5 minutes ago). Optional; omit to restore to each snapshot's creation time.
```

Comment:
oh, I didn't know you can omit the time? is that so?

Reply:
We CANNOT omit the time. But there are 2 ways to specify it:
1) absolute time. For example: 2021-06-15T11:13:00
2) relative time, which should be prefixed with "minus" word. I.e. For example: minus 10m

Reply:
Thank you, Sergei! Updated.